networking/network_security/AdminNetworkPolicy/ovn-k-anp-banp-metrics.adoc

```diff
@@ -0,0 +1,11 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ovn-k-anp-banp-metrics"]
+= Monitoring ANP and BANP
+include::_attributes/common-attributes.adoc[]
+:context: ovn-k-anp-banp-metrics
+
+toc::[]
+
+`AdminNetworkPolicy` and `BaselineAdminNetworkPolicy` have metrics that can be used for monitoring and managing your policies. See the below table for more details on the metrics.
```

Comment:
Same question of if you can qualify these - usually anytime we write about API objects or whatnot, we want to say what it is afterward

```suggestion
`AdminNetworkPolicy` and `BaselineAdminNetworkPolicy` resources have metrics that can be used for monitoring and managing your policies. See the following table for more details on the metrics.
```